src/utils/transformErrors.ts

```diff
@@ -60,6 +60,7 @@ const dateErrorHandler = (error: Joi.ValidationError): FormFieldError | undefine
   let toError: string = '';
   const toItems: GovUKItems[] = [];
   Object.keys(error._original).forEach((field) => {
+    console.log(field);
```

Comment:
remove this

Reply:
I apologize, I have removed this console statement now.